site/app/templates/course/CourseMaterials.twig

```diff
@@ -221,46 +225,14 @@
 
     window.onload = function () {
```

Comment:
Same comment for the body of this function as well.

Reply:
Serving "dead" code to lower-access users for the sake of maintainability is fine in my opinion.  Static JS files are often cached by browsers, and are negligible in size compared to media, so there should be no performance hit associated with the extra bytes being transmitted.

Reply:
That seems reasonable to me.  Please make an issue so we don't forget to do it.